lib/glific/flows/webhook.ex

```diff
@@ -181,15 +181,40 @@ defmodule Glific.Flows.Webhook do
     nil
   end
 
+  # THis function will create a dynamic headers
+  @spec create_headers(Action.t(), FlowContext.t()) :: map()
+  defp create_headers(action, context) do
+    default_payload = %{
+      contact: %{
+        id: context.contact.id,
+        name: context.contact.name,
+        phone: context.contact.phone,
+        fields: context.contact.fields
+      },
+      results: context.results
+    }
+
+    fields = %{
+      "contact" => Contacts.get_contact_field_map(context.contact_id),
+      "results" => context.results,
+      "flow" => %{name: context.flow.name, id: context.flow.id}
+    }
+
+    MessageVarParser.parse_map(action.headers, fields)
```

Comment:
Can you explain why we need this ?

Reply:
@pankaj-ag can you point out? Not sure what you are referring to here?

Reply:
Line number 204 - 210 in this file.